incubator/elasticsearch/README.md

```diff
@@ -0,0 +1,65 @@
+# Elasticsearch Helm Chart
+
+This image is using Fabric8's great work around the [kubernetes plugin](https://github.com/fabric8io/elasticsearch-cloud-kubernetes) for elasticsearch and their [image](https://hub.docker.com/r/fabric8/elasticsearch-k8s/) as parent.
```

Comment:
This is not a workaround, but rather a cloud plugin.  There image is not the latest elastic search btw.